README.md

```diff
@@ -29,13 +29,14 @@ func main() {
 ```
 
 Build the program with the [go tool](https://golang.org/cmd/go/) to an
-executable named `hello`. Ensure that the executable is on your path.
+executable named `hello`. Ensure that the executable is in a directory in
+the `PATH` environment variable.
 
 ```go
-// Use the go build to generate an executable
-// Enusre this hello executable on your path
-// you can move hello to your $GOPATH/bin
-// or set the current dir into env variable `path`
+// Use the `go build` command to generate an executable.
+// Ensure that this "hello" executable is on your path.
```

Comment:
`that this ` ?

Reply:
What drew my attention was the spelling `Enusre`, but decided to restructure some others. I could close if you don't think it's worth fixing.

Reply:
Sorry, I think I see your issue, has it been fixed with the latest commit?